modules/Sources/Features/NHSendFlow/NHSendFlowStore.swift

```diff
@@ -0,0 +1,268 @@
+//
+//  NHSendFlowStore.swift
+//  
+//
+//  Created by Matthew Watt on 7/19/23.
+//
+
+import ComposableArchitecture
+import DerivationTool
+import MnemonicClient
+import SDKSynchronizer
+import SwiftUI
+import Utils
+import WalletStorage
+import ZcashLightClientKit
+import ZcashSDKEnvironment
+
+public typealias NHSendFlowStore = Store<NHSendFlowReducer.State, NHSendFlowReducer.Action>
+public typealias NHSendFlowViewStore = ViewStore<NHSendFlowReducer.State, NHSendFlowReducer.Action>
+
+public struct NHSendFlowReducer: ReducerProtocol {
+    private enum SyncStatusUpdatesID { case timer }
+    let networkType: NetworkType
```

Comment:
[non-blocking] 
Shouldn't this be a `@dependency` instead? It might be that because of developer error the wrong value lands here and it will be tough to debug.

Reply:
It could be. Again,  I'll note this as a future enhancement